manifests/base/deployment.yaml

```diff
@@ -26,6 +24,6 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
-        image: gcr.io/kubeflow-images-public/tf-operator:v0.6.0
+        image: public.ecr.aws/j1r0q0g6/training/tf-operator
```

Comment:
CI already cover image build to public ECR?

Reply:
CI push to private ECR
CD push to public ECR

The registry is defined differently in the prow_config.yaml file

Reply:
Sounds good